src/status.rs

```diff
@@ -278,13 +276,17 @@ impl Deserialize for Status {
         };
 
         impl Deserialize for Field {
-            fn deserialize<D: Deserializer>(deserializer: &mut D) -> Result<Field, D::Error> {
+            fn deserialize<D: Deserializer>(deserializer: D) -> Result<Field, D::Error> {
                 struct FieldVisitor;
 
                 impl Visitor for FieldVisitor {
                     type Value = Field;
 
-                    fn visit_str<E: SerdeError>(&mut self, value: &str) -> Result<Field, E> {
+                    fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+                        write!(formatter, "a valid field name")
```

Comment:
I don't think this error message can be triggered, since the only way the visit_str below can fail, is if the field is unknown. Also the messages would fit the template from https://docs.serde.rs/serde/de/trait.Visitor.html#tymethod.expecting "This Visitor expects to receive ... a valid field name".